Tests/Unit Tests/Conditions/PostcodeConditionTests.swift

```diff
@@ -14,189 +14,96 @@ import XCTest
-        
-        // Test
-        AssertCondition(conditionSWE, testInput: testInput, expectedResult: expectedResult)
-    }
-    
-    func testPostcodeCondition_SWE_WithoutGap_Success() {
-        // Given
-        let testInput       = "11434"
-        let expectedResult  = true
-        
-        // Test
-        AssertCondition(conditionSWE, testInput: testInput, expectedResult: expectedResult)
-    }
-    
-    func testPostcodeCondition_SWE_International_Default_Success() {
-        // Given
-        let testInput       = "SE-111 21"
-        let expectedResult  = true
-        
-        // Test
-        AssertCondition(conditionSWE, testInput: testInput, expectedResult: expectedResult)
-    }
-    
-    func testPostcodeCondition_SWE_International_WithoutGap_Success() {
-        // Given
-        let testInput       = "SE-11637"
-        let expectedResult  = true
-        
-        // Test
-        AssertCondition(conditionSWE, testInput: testInput, expectedResult: expectedResult)
-    }
-    
-    func testPostcodeCondition_SWE_International_LowerCase_Success() {
-        // Given
-        let testInput       = "se-11637"
-        let expectedResult  = true
-        
-        // Test
-        AssertCondition(conditionSWE, testInput: testInput, expectedResult: expectedResult)
+    // MARK: - Configuration
+    
+    private struct TestableCondition {
+        
+        let country: PostcodeCountries
+        let condition: PostcodeCondition
+        
+        var successInputs: [String] {
+            switch country {
+            case .Sweden:
+                return ["112 50",
+                        "11434",
+                        "SE-111 21",
+                        "SE-11637",
+                        "se-11637"]
+            case .Turkey:
+                return ["34345"]
+            case .UnitedKingdom:
+                return ["M1 1BA"]
+            case .UnitedStates:
+                return ["20500",
+                        "95014-2083"]
+            }
+        }
+        
+        var failureInputs: [String?] {
+            switch country {
+            case .Sweden:
+                return ["113 4",
+                        "116233",
+                        "us-125 41",
+                        "us-125e1",
+                        nil]
+            case .Turkey:
+                return ["3411",
+                        "347001",
+                        "34 700",
+                        "3470a",
+                        nil]
+            case .UnitedKingdom:
+                return ["M1AA 1BA",
+                        "M1 1BAA",
+                        nil]
+            case .UnitedStates:
+                return ["1234",
+                        "12345-1",
+                        nil]
+            }
+        }
+        
+        init(country: PostcodeCountries) {
+            self.country = country
+            condition = PostcodeCondition(country: country)
+        }
+        
```

Comment:
Convention in this project has been to have an empty line between the end of the last function/property/etc. and the ending closing brace of the class/struct/enum/etc.